src/elli/importer/accurion.py

```diff
@@ -0,0 +1,99 @@
+"""A helper class to load data from Accurion EP4 DAT files.
+Typical files look like: Si3N4_on_4inBF33_W03_20240905-105631.ds.dat
+"""
+
+import pandas as pd
+import chardet
+import numpy as np
+
+def detect_encoding(fname: str):
+	r"""Detects the encoding of file fname. Used in read_accurion_psi_delta only.
+  Args:
+    fname (str): Filename
+  """
```

Comment:
Here is something odd with indentation. However, ruff formatting will fix this for you (see previous comment).